media-tv/mythtv/mythtv-32.0.ebuild

```diff
@@ -0,0 +1,427 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
8?

Reply:
EAPI=8 works with no modifications to ebuild.

Reply:
Please force push with these changes